build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
     implementation 'org.springframework.boot:spring-boot-starter-webflux'
+    implementation group: 'org.jsoup', name: 'jsoup', version: '1.7.2'
```

Comment:
Do you think it's better than HtmlUnit library we already have?

Reply:
I can't say HtmlUnit is easier, but I switched to it